### ISSUE TITLE
Memory reservation not passed to task definition

### ISSUE THREAD

**Issue by c100k** [aws/amazon-ecs-cli#262]
Hi all,

I thought it was ready to work after reading : https://github.com/aws/amazon-ecs-cli/issues/145
But I still have the following warning and the value is not set in the task definition :
```
time="2017-06-16T04:11:30Z" level=warning msg="Skipping unsupported YAML option for service..." option name="mem_reservation" service name=api 
```

Compose file :

```yaml
version: '2'
services:
  api:
    image: "myImage"
    mem_reservation: 512m
```

Configuration :

```
host:~$ docker -v
Docker version 1.13.1, build 092cba3
host:~$ docker-compose -v
docker-compose version 1.13.0, build 1719ceb
host:~$ ecs-cli -v
ecs-cli version 0.5.0 (1aa8028)
```

Am I missing something ?
Thanks

**Comment by yinshiua** [comment]
Hi, this will be included in the next release. You can get the latest by downloading the package and build your own binary.

**Comment by jpeterson-isdc** [comment]
When will the next release be tagged?

**Comment by yinshiua** [comment]
We are looking at doing a release this week.

**Comment by yinshiua** [comment]
Thanks for your patience. This is released in version 0.6.0.
